src/main/kotlin/com/kotlin/study/dongambackend/domain/comment/repository/CommentQueryDslRepository.kt

```diff
@@ -0,0 +1,49 @@
+package com.kotlin.study.dongambackend.domain.comment.repository
+
+import com.kotlin.study.dongambackend.domain.comment.entity.Comment
+import com.kotlin.study.dongambackend.domain.comment.entity.QComment
+import com.kotlin.study.dongambackend.domain.post.entity.Post
+import com.querydsl.core.types.dsl.BooleanExpression
+import com.querydsl.jpa.impl.JPAQueryFactory
+import org.springframework.data.domain.*
+import org.springframework.stereotype.Repository
+
+@Repository
+class CommentQueryDslRepository(val queryDslFactory: JPAQueryFactory) {
+    val qComment = QComment.comment;
+
+    fun searchCommentsBySlice(lastCommentId: Long, pageable: Pageable): Slice<Comment> {
+        val results: List<Comment> = queryDslFactory.selectFrom(qComment)
+            .where(
+                qComment.isDeleted.eq(false)
+            )
+            .orderBy(qComment.id.desc())
+            .limit((pageable.pageSize + 1).toLong())
+            .fetch() as List<Comment>
+
+        // 무한 스크롤 처리
+        return checkLastPage(pageable, results)
+    }
+
+    // no-offset 방식 처리하는 메서드
+    private fun ltCommentId(commentId: Long): BooleanExpression? {
```

Comment:
이 부분 아까 경고 발생하더라고요! 아래 처럼 수정하셔야할 것 같습니다. (suggestion 반영 주의..)
```suggestion
    private fun ltCommentId(commentId: Long?): BooleanExpression? {
```

Reply:
아 이부분은 확인하지 못했네요! 수정해두겠습니다